app/manage/lib/core/model/user_profile_model.dart

```diff
@@ -0,0 +1,13 @@
+import 'dart:ui';
+import 'package:flutter/cupertino.dart';
+import 'package:manage/core/model/manage_model.dart';
+
+class UserProfileModel extends ManageModel{
```

Comment:
UserProfileModel is not JSON serialized. You can serialize this easily by looking to other models.
We also need a TeamUserModel for other users as you can check it out at swagger doc. Although it will be merged to develop soon, you can still implement it yourself to not wait.